topi/python/topi/nn/conv2d.py

```diff
@@ -329,7 +329,64 @@ def conv2d_NCHWc(data, kernel, stride, padding, dilation, layout, out_layout, ou
     """
     # search platform specific declaration first
     # default declaration
-    raise ValueError("missing register for topi.nn.conv2d_NCHWc")
+    # layout and out_layout are not used here,
+    # we keep them for debug convenience when dumping autotvm workload
+    HPAD, WPAD = padding if isinstance(padding, (tuple, list)) else (padding, padding)
```

Comment:
Could we use `   pad_top, pad_left, pad_down, pad_right = get_pad_tuple(
        padding, (dilated_kernel_h, dilated_kernel_w))` then `HPAD = pad_top + pad_down` and `WPAD = pad_left + pad_right`? This would make this RFC: https://github.com/dmlc/tvm/issues/2682 easier.